src/App.tsx

```diff
@@ -31,11 +27,31 @@ class App extends Component<Props, State> {
     });
   }
 
+  getNewCall = async (selection: string) => {
+    console.log(selection);
+    if (selection === "compliment") {
+      console.log("inside compliment");
```

Comment:
I loved how we made this dynamic! We'll have to remember to removed those console logs!